pkg/verifier/notation/notation.go

```diff
@@ -176,12 +177,38 @@ func (v *notationPluginVerifier) Verify(ctx context.Context,
 	return verifier.NewVerifierResult("", v.name, v.verifierType, "Notation signature verification success", true, nil, extensions), nil
 }
 
-func getVerifierService(conf *NotationPluginVerifierConfig, pluginDirectory string) (notation.Verifier, error) {
+func getVerifierService(conf *NotationPluginVerifierConfig, pluginDirectory string, f RevocationFactory) (notation.Verifier, error) {
 	store, err := newTrustStore(conf.VerificationCerts, conf.VerificationCertStores)
 	if err != nil {
 		return nil, err
 	}
-	verifier, err := notationVerifier.New(&conf.TrustPolicyDoc, store, NewRatifyPluginManager(pluginDirectory))
+
+	// revocation check using corecrl from notation-core-go and crl from notation-go
+	// This is the implementation for revocation check from notation cli to support crl and cache configurations
```

Comment:
Hi @junczhu , do you mean starting from 188 is the actual impl from notation cli? could you add a link to the repo?

Reply:
Added in the description. Just tiny changes that help further CRL related implementation.
cc\ @susanshi

Reply:
Hi @junczhu , would you be add the link to the source code? when this piece of code is reviewed, it will be helpful to see a link in the src code  without requiring to pull out the pr description :)